azurerm/internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -87,9 +87,37 @@ func resourceArmKustoCluster() *schema.Resource {
 						},
 
 						"capacity": {
+							Type:          schema.TypeInt,
+							Optional:      true,
```

Comment:
Will be changed to Optional and Computed, as the value depends on the optimized auto scale settings if optimized auto scale is enabled.